src/store/plugins/walletsPlugin.js

```diff
@@ -411,6 +419,47 @@ class WalletsPlugin {
       isSimplified ? { walletId } : {},
     )
   }
+
+  requestETHBalanceByAddress = async (address: Address): Promise<string> => {
+    const network: Network = this.getNetwork()
+
+    try {
+      const ethBalance: string = await web3.getAssetBalance(network, address, 'Ethereum')
+
+      return new BigNumber(ethBalance)
```

Comment:
This will re-request balance when server answers with invalid value and it can not be converted to BigNumber.
I think, it need to re-request data only when network error occurred. When server answers with 200 responce, but value is invalid we should set error for this balance value

Reply:
Do you know case when answer code is 200 and value is not valid bignumber?